### DISCUSSION TITLE
Github Education Application - Rejected Multiple Times

### DISCUSSION THREAD

**Discussion by iulianerc** [community/community#141922] [GitHub Education]
### Hi everyone,

Hello, I tried recently to submit a request for github education but it failed multiple times.
Each time it fails because of the document prof I uploaded for confirmation.
Below is the email I received: 
```
Hey iulianerc,

Unfortunately, we were unable to verify your academic affiliation based on the information you submitted on Oct 18, 2024 for [@iulianerc](https://github.com/iulianerc).

    Attendees of this school must verify and use their academic email address. When you have verified your school's academic email address on your account, ensure that you select the school when prompted on the application page.

    Please use your device camera to submit academic affiliation documents.

    Please ensure that your academic affiliation document contains your first name exactly as it appears in your [GitHub billing information](https://github.com/settings/billing/payment_information). Variations of your name and nicknames are unacceptable.

    You are unlikely to be verified until you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name exactly as it appears in your academic affiliation document. Please do not use a variation of your name or a nickname. Once you have updated your profile information log out and log back into GitHub before re-applying.


Once you have resolved the issues with your submission, [visit us again and reapply](https://education.github.com/).

Have questions? Please seek help via our[ Student Developer Pack Application FAQ ](https://github.com/orgs/community/discussions/111352)or visit the GitHub Education Community.

💖 The GitHub Education Team
```

In the point of affiliation document, it says something about my First and Last name, 
it has to be like in the document, but _my student book has these fields handwritten_ 

Could this be the reason why it was rejected?
Help please, I don't know what to do `(

**Comment by marlonwq** [comment]
Did you use academic email?

**Reply by MG72021** [reply]
[google.com](url)

**Reply by iulianerc** [reply]
I would in the first place, but don't have one, if no solution Is found, then will just try to request an academic email from my university

**Reply by marlonwq** [reply]
I understand, I was only able to redeem mine after checking the academic email in the github account and filling out the github education request using it...

**Comment by fjyeo** [comment]
Hi mate,

The issue might be that your handwritten name in the student book doesn't exactly match your GitHub profile. I would maybe try:

1. Use your verified academic email.
2. Make sure your GitHub name matches your document exactly.
3. Try submitting a clearer document, if possible.
4. Double-check your profile, then log out and back in before reapplying.

Good luck!

**Reply by iulianerc** [reply]
Ok, thank you
I didn't even think about re-logging after changes in my profile before reapplying.

And one more question: 
1. Is there a difference between the first and last name order in my profile?
2. If my document is not in English would it still work, or do I have to translate that somehow (I found something about this case in some YouTube video)?

**Reply by fjyeo** [reply]
You're welcome! 

For the name order, it's best to match it exactly as in your document. to avoid confusion. If your document isn’t in English, submitting both the original and a translation might help things go smoother.

**Reply by BossHoder** [reply]
My school does not provide edu mail.
Hey, I'm having a bit of trouble with GitHub's billing info. I've tried entering my name as "TRƯƠNG THẾ ANH" and "THẾ ANH TRƯƠNG," but it's not working. I'm applying for the education pack using my student ID from Vinh University of Technical Education. I made sure the picture I uploaded shows the whole front side of my ID, but it keeps saying there's an issue. Any ideas on what I can do? 
"Hey BossHoder,

Unfortunately, we were unable to verify your academic affiliation based on the information you submitted on Nov 1, 2024 for [@BossHoder]

Please ensure that your academic affiliation document contains your last name exactly as it appears in your [GitHub billing information]If you have only a single legal name, enter it in both the first and last name fields of your GitHub billing information.

Please ensure that your academic affiliation document contains your first name exactly as it appears in your [GitHub billing information]. Variations of your name and nicknames are unacceptable.

You are unlikely to be verified until you have completed your [GitHub user profile] with your full name exactly as it appears in your academic affiliation document. Please do not use a variation of your name or a nickname. Once you have updated your profile information log out and log back into GitHub before re-applying.



Once you have resolved the issues with your sumbission, [visit us again and reapply]

Have questions? Please seek help via our [Student Developer Pack Application FAQ 

💖 The GitHub Education Team"

**Comment by naryth** [comment]
I used to apply for the GitHub Student Pack with my university email and student card, but I have also been rejected.
